README.md

```diff
@@ -1 +1,39 @@
-# diesel-adapter
\ No newline at end of file
+# diesel-adapter
+
+An adapter designed to work with [casbin-rs](https://github.com/casbin/casbin-rs).
+
+
+## Install
+
+Add it to `Cargo.toml`
+
+```
+diesel_adapter = { version = "0.1.0", features = ["postgres"] }
+```
+
+
+## Example
+
+```rust
+extern crate casbin;
```

Comment:
No need to write `extern crate` for rust 2018 edition, just write `use casbin::{Model, Adapter}`